File: data/code_pairs/cpp/1577.cpp
#include <vector>
#include <numeric>
#include <iostream>

class MEANFLUXES__genmod {
public:
    static double MEANFLUXES(const std::vector<double>& fluxes) {
        if(fluxes.empty()) return 0.0; // Handle empty vector case
        double sum = std::accumulate(fluxes.begin(), fluxes.end(), 0.0);
        return sum / fluxes.size();
    }
};

File: data/code_pairs/fortran/1577.f90
MODULE MEANFLUXES__genmod
  IMPLICIT NONE
CONTAINS
  SUBROUTINE MEANFLUXES(fluxes, n, mean_flux)
    REAL, INTENT(IN) :: fluxes(:)
    INTEGER, INTENT(IN) :: n
    REAL, INTENT(OUT) :: mean_flux
    mean_flux = SUM(fluxes) / REAL(n)
  END SUBROUTINE MEANFLUXES
END MODULE MEANFLUXES__genmod